form/multiple_buttons.rst

```diff
@@ -31,3 +31,10 @@ querying if the "Save and add" button was clicked::
 
         return $this->redirectToRoute($nextAction);
     }
+
+.. caution::
+
+    When the user submits your form by pressing the ``Enter`` key on the keyboard,
+    some browsers do include the submit button field in the form data, and
+    some don't. So you should take care of handling the case when ``isClicked()``
```

Comment:
Hmm, reading this note I understand that in all my apps I should take care the the user agent header before performing the check...

Not sure we should care.